react/react1/week3/todo-app3/src/components/ResultData.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext, useEffect } from 'react';
+import { UserContext } from './Context';
+
+function ResultData() {
+  const { list, setList, input, error, setError, isLoading, setIsLoading } =
```

Comment:
Hi Anna @annamourasobral .
Vie did put all states in context, because it is the task)
![image](https://user-images.githubusercontent.com/73549460/198265771-502b1f17-d530-479e-9943-78c6eb8f2f3c.png)